python/GafferSceneUI/InstancerUI.py

```diff
@@ -345,12 +345,9 @@ def __init__( self, headings, toolTipOverride = "" ) :
 
 			"description",
 			"""
-			The object on which to make the instances. The
-			position, orientation and scale of the instances
-			are taken from per-vertex primitive variables on
-			this object. This is ignored when a filter is
-			connected, in which case the filter specifies
-			multiple objects to make the instances from.
+			Using the `parent` plug to select the source is now obselete, please use a filter instead.
```

Comment:
obselete -> obsolete. Should we explicitly deprecate it though instead?

Reply:
What would explicitly deprecating it mean in this case? This plug is already hidden by the base class unless there's a non-default value on it, so no one is seeing it except in legacy scenes.

Are you suggesting we should actually remove the plug? That does sound worthwhile at some point, though I guess maybe I just missed the 1.5 window?

Reply:
For now, I've just fixed the typo.

Reply:
I was just thinking of using the word "deprecated" rather than "obsolete" - I think that implies we might remove it one day. (Definitely not suggesting we remove it now - it's useful still for the Parent node at least).